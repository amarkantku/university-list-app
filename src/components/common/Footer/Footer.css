.footer {
    display: flex;
    justify-content: center;
    align-items: center;
    height: 10vh;
    width: 100%;
    background: #ddd;
    color: rgb(61, 60, 60);
}