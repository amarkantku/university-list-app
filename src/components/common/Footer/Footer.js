import React from 'react'

import classNames from 'classnames/bind';
import styles from './Footer.css';
let clx = classNames.bind(styles);

function Footer() {
    return (
        <footer data-testid='footer'>
            <div className={clx('footer')}>
                Footer
            </div>
        </footer>
    )
}

export default Footer;
