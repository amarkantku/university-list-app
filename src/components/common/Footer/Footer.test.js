import React from 'react';
import Footer from './Footer';
import { render } from '@testing-library/react';

describe('<Footer />', () => {
  test('should render correct text', () => {
    const { asFragment, queryByText } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
    expect(queryByText('Footer')).toBeTruthy();
  });
  
  test('should have footer id element', () => {
      const { getByTestId } = render(<Footer />);
      expect(getByTestId('footer')).toBeTruthy();
  });
})
