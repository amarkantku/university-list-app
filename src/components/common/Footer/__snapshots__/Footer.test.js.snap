// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render correct text 1`] = `
<DocumentFragment>
  <footer
    data-testid="footer"
  >
    <div
      class="footer"
    >
      Footer
    </div>
  </footer>
</DocumentFragment>
`;
